schema-engine/connectors/sql-schema-connector/src/sql_schema_differ/differ_database.rs

```diff
@@ -312,6 +325,30 @@ impl<'a> DifferDatabase<'a> {
         })
     }
 
+    fn is_table_external(&self, table: &TableWalker<'_>) -> bool {
+        // TODO:(schema-filter) optimize for speed to avoid recomputing the underlying contains?
+        self.filter.is_table_external(table.namespace(), table.name())
+    }
+
+    fn is_table_pair_external(&self, tables: MigrationPair<TableWalker<'_>>) -> bool {
+        if self.is_table_external(&tables.next) ^ self.is_table_external(&tables.previous) {
```

Comment:
Nice XOR lol

Reply:
Was waiting for this moment to actually use it in production since my study times! 😂